### PR TITLE
fix(origin-ui) Fix smart meter reads and allowing certificates bulk t…

### PR DESCRIPTION
**Pull request by Malak67**
Fixing smart meter reads
Allowing bulk transfer and claim of certificates

### CODE REVIEW

File: packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.effects.ts [hunk 1]
```diff
@@ -36,9 +37,10 @@ export const useCertificateActionContentEffects = <Id>(
               items.find((item) => item.id === id).energy.replace(/,/g, '')
             ).toString(),
           };
-          setEnergyAmounts([...energyAmounts, preparedNewItem]);
+          newItems.push(preparedNewItem);
         }
       });
+      setEnergyAmounts([...energyAmounts, ...newItems]);
```
Comment (alexworker23) [inline packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.effects.ts:43]: why do you change this logic? did you encounter any bug related to selection of items and this is the fix?

File: packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.tsx [hunk 2]
```diff
@@ -107,13 +100,6 @@ export const CertificateActionContent: TCertificateActionContent = ({
           </>
         )}
       </Button>
-      {selectedIds.length > 1 && (
```
Comment (alexworker23) [inline packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.tsx:110]: Have you checked if the bulk actions work with all the available actions of both Exchange and Blockchain inboxes ?

**Review by alexworker23 — CHANGES_REQUESTED** [review]
`index.html` is auto-generated file and should not be included

**Comment by alexworker23** [comment]
Also please follow our conventional-commit rules and commit changes per package to keep our changelogs updated accordingly

File: packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.effects.ts (hunk 1 above)
Comment (Malak67) [reply packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.effects.ts:43]: Yes. When selecting multiple at once, only the value for the last one is taken into account. This was the fix for it.

File: packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.tsx (hunk 2 above)
Comment (Malak67) [reply packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.tsx:110]: I have tested this for Blockchain inbox (locally with D-REC) cannot test in Origin. Need your help here.

Comment (alexworker23) [reply packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.tsx:110]: Ok, will check it

**Comment by Malak67** [comment]
> Also please follow our conventional-commit rules and commit changes per package to keep our changelogs updated accordingly

Can you give me an example on how to rename the commits?

**Comment by alexworker23** [comment]
> Can you give me an example on how to rename the commits?

#3235 for example

**Comment by Malak67** [comment]
> `index.html` is auto-generated file and should not be included

Initially I did not add it but I noticed it failed the Ling & Test because of it. I will remove it again.

**Comment by alexworker23** [comment]
Here is how it looks like:
- Exchange Inbox
1. Sell - not working as bulk actions only posts the first one for sale
2. Withdraw - doesn't work at all with multiple certificates
3. Transfer - behaves weirdly. Transfers the first certificate to user's blockhain address (behaves as withdrawal) and the last one to the actual address mentioned in inputs. So we should probably disable bulk operations for this action for now.
4. Export - unable to properly test, but for the safety of operations lets also assume it couldn't handle multiple 
5. Claim - claims only the first one, for the second one errors. 

- Blockchain Inbox
1. Deposit - works fine
2. Retire - works fine 
3. Transfer - works fine

So I would say that we should allow bulk actions only for Blockchain Inbox

**Comment by alexworker23** [comment]
You have to have 3 commits min:
1 - certificate
2 - device
3 - localization

**Review by alexworker23 — APPROVED** [review]

**Pull request merged**